src/recorder/__tests__/HttpRecorder.test.ts

```diff
@@ -122,3 +124,35 @@ describe('HttpRecorder', () => {
     expect(testCassette[0].response).to.equal(TEST_RESULT);
   });
 });
+
+describe('Restrict Content by File', () => {
+  before(() => {
+    server = startServer();
+  });
+
+  after(() => {
+    server.close();
+  });
+
+  afterEach(() => {
+    if (fs.existsSync(TEST_CASSETTE_DUPLICATE)) {
+      fs.unlinkSync(TEST_CASSETTE_DUPLICATE);
+    }
+  });
+  
+  it('should create a file and write content without previous tests', async () => {
```

Comment:
just one minor thing, we are trying to have one `expect` per test regardless of having the same scenario